jhove-modules/pdf-hul/src/main/resources/edu/harvard/hul/ois/jhove/module/pdf/ErrorMessages.properties

```diff
@@ -149,15 +149,18 @@ PDF-HUL-144 = Pages dictionary has no Type key or it has a null value.
 PDF-HUL-145 = Pages dictionary Type key does not have a simple String value.
 PDF-HUL-146 = Pages dictionary Type key must have value /Pages.
 PDF-HUL-147 = Page tree node not found.
-PDF-HUL-148 = PDF minor version number is greater than 7.
+PDF-HUL-148 = PDF minor version number is out of valid range.
 PDF-HUL-149 = Invalid indirect destination - referenced object ''{0}'' cannot be found
 PDF-HUL-150 = Cross-reference stream must be a stream
 PDF-HUL-151 = Unexpected error occurred while attempting to read the cross-reference table
 PDF-HUL-152 = Encrypt dictionary has no OE key or it has a null value
 PDF-HUL-153 = Encrypt dictionary has no UE key or it has a null value
 PDF-HUL-154 = Unknown Developer Prefix:
-PDF-HUL-155 = Error parsing mandatory version number from PDF header.
+PDF-HUL-155 = Error parsing mandatory version number, PDF header too short: "{0}"
 PDF-HUL-156 = Extension can't be defined as an indirect reference
 PDF-HUL-157 = Unexpected exception {0}
 PDF-HUL-158 = Unexpected exception {0}
```

Comment:
Hmm the wiki needs to be updated :-) https://github.com/openpreserve/jhove/issues/967